zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -1052,7 +1051,7 @@ private void createNote(NotebookSocket conn, HashSet<String> userAndRoles, Noteb
       note.persist(subject);
       addConnectionToNote(note.getId(), (NotebookSocket) conn);
       conn.send(serializeMessage(new Message(OP.NEW_NOTE).put("note", note)));
-    } catch (FileSystemException e) {
+    } catch (IOException e) {
```

Comment:
Changing this to IOException as Hadoop throws org.apache.hadoop.security.AccessControlException

Reply:
LGTM